src/components/AddTaskField.js

```diff
@@ -1,24 +1,52 @@
 import { useForm } from "react-hook-form";
-import { Button, Box, TextField } from "@mui/material";
+import { Box } from "@mui/material";
+import Buttons from "components/TodoButtons";
+import Input from "components/TodoInputs";
+import { makeStyles } from "@mui/styles";
+const useStyles = makeStyles({
+  todoBtn: {
+    color: "white",
+    width: "28%",
+    border: "1px solid white",
+    height: "56px",
+    marginTop: "auto",
+    textAlign: "center",
+    marginLeft: "71%",
```

Comment:
Try to not to give one sided margins or paddings in all over the platform, instead manage it by using Mui flex, grids and other properties or you can give margins on both side but one sided margins should be removed